src/EA.Weee.Web/Areas/Aatf/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -149,7 +203,6 @@ private void SetSuccessMessage(EvidenceNoteData note, ViewEvidenceNoteViewModel
                     model.SuccessMessage = (status == NoteStatus.Submitted ?
                         $"You have successfully submitted the evidence note with reference ID E{note.Reference}" : $"You have successfully saved the evidence note with reference ID E{note.Reference} as a draft");
 
-                    //TODO: Move this to the mapper
```

Comment:
Was moving this to the mapper not required?

Reply:
Yes and looks like I didnt move it when removing the comment. Will update as part of the tests.